app/spec_helpers/client_reporter.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+class ClientReporter {
+
+  constructor() {
+    this.buildPassed = true;
+    this.emitter = false;
+  }
+
+  setEmitter(emitter) {
+    this.emitter = emitter;
+  }
+
+  specDone(result) {
+    if (this.emitter) {
+      this.emitter.specDone(result);
+    } else {
+      setTimeout(() => {
+        this.specDone(result);
+      }, 100);
```

Comment:
This pattern is used in both methods, maybe there's a better way? Maybe a proxy type of decorator?